saleor/graphql/account/utils.py

```diff
@@ -108,8 +108,8 @@ def can_manage_app(requestor: Union["User", "App", None], app: "App") -> bool:
 def get_group_permission_codes(group: Group) -> "QuerySet":
     """Return group permissions in the format '<app label>.<permission codename>'."""
     return group.permissions.annotate(
-        formated_codename=Concat("content_type__app_label", Value("."), "codename")
-    ).values_list("formated_codename", flat=True)
+        formatted_codename=Concat("content_type__app_label", Value("."), "codename")
+    ).values_list("formatted_codename", flat=True)  # type: ignore[misc]
```

Comment:
There are still some usages of `formated_codename` in the codebase; maybe we should fix all of them on this occasion. I'm wondering if changing it only here doesn't break it somewhere else where `formated_codename` is expected.